README.md

```diff
@@ -44,6 +44,9 @@ This script supports several flags, all of which are optional.
 * `--mkfs_options`  
   Extra options to pass to `mkfs`. Useful for settings bytes per inode on ext4,
   e.g. `--mkfs_options="-i 65536"`.
+* `--mount_options`  
+  Extra options to pass to `mount`. Useful for settings compression algorithm on btrfs.
+  e.g. `--mount_options="-o compress=lzo"`.
```

Comment:
I feel that making `mount_options` not take `-o` might make more sense (`--mount-options="compress=lzo,noatime"`) and avoids users from passing in other extraneous flags.